frontend/src/components/ui/blockEditor/CodeMirrorComponents.jsx

```diff
@@ -68,14 +71,23 @@ export const ViewerCodeMirror = ({ code }) => {
 
 export const EditorCodeMirror = ({ code, onChange }) => {
   const [theme] = useAtom(themeAtom);
+  const extensions = [loadLanguage("python"), viewerBackgroundTheme];
```

Comment:
```suggestion
  const extensions = [loadLanguage("python"), backgroundTheme];
```
@PatrickStAmant specifically asked to make the editor block darker than the viewer blocks, see: #103. I think it's the same color in light mode though.